src/src/rx_main.cpp

```diff
@@ -1388,7 +1389,7 @@ static void setupRadio()
         return;
     }
 
-    POWERMGNT::setPower((PowerLevels_e)config.GetPower());
+    DynamicPower_UpdateRx(true);
```

Comment:
this `true` here, doesn't convey any information and forces the reader to look at the implementation of `DynamicPower_UpdateRx` to find out what it does, which means reading and understanding the code takes longer too.